### ISSUE TITLE
configuration file cannot be read

### ISSUE THREAD

**Issue by yamap55** [RobertOstermann/vscode-sqlfluff-extended#2]
configuration file cannot be read.
Where is the default "sqllfluff.format.workingDirectory"?

I expected the root of the project, but it seems different.

## error

```
Error loading config: Extra config '.sqlfluff' does not exist.
```

## settings.json

```json
    "sqlfluff.dialect": "bigquery",
    "sqlfluff.linter.run": "onSave",
    "sqlfluff.config": ".sqlfluff",
```

## directory tree

```
/
├ sql
│  └ a.py <- format target file
└ .sqlfluff
```

## temporary solution

```json
    "sqlfluff.dialect": "bigquery",
    "sqlfluff.linter.run": "onSave",
    "sqlfluff.config": ".sqlfluff",
    "sqlfluff.format.workingDirectory": "d:/github/repository_name" <- new!
```

**Comment by RobertOstermann** [comment]
@yamap55 That was a mistake on my part. I think the default ended up being the path to the file you were working in. But it definitely should default to the root path of the project. I have fixed this bug in version 0.1.7

**Comment by yamap55** [comment]
@RobertOstermann Thank you !!
I was able to confirm that it works correctly.

I have found this extension to be very helpful.
Thanks for developing it.
